deepspeed/pt/deepspeed_zero_optimizer.py

```diff
@@ -1230,6 +1492,21 @@ def unscale_and_clip_grads(self, grad_groups_flat, norm_groups):
             else:
                 grad.data.mul_(1. / combined_scale)
 
+    def unscale_and_clip_grads_on_cpu(self, grad_groups_flat, norm_groups):

```

Comment:
I think this method is equivalent to calling
```python
self.unscale_and_clip_grads((self.single_partition_of_fp32_groups.grad, ), norm_groups)
```

By wrapping the single grad in a tuple of size 1 the original code will do the same thing.